### PR TITLE
add Translation for Korean

### PR DESCRIPTION
**Pull request by kangsLee**
Added Korean translation.

## Description

Added Korean translation.
I have modified the Readme.md and Gruntfile.js files.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: static/locales/ko.json [hunk 1]
```diff
@@ -0,0 +1,755 @@
...
+    "Wurmple": "개무소",
+    "Wynaut": "마자",
+    "Xatu": "네이티오",
+    "Xerneas": "제르네아스",
+    "Yamask": "데스마스",
+    "Yanma": "왕자리",
+    "Yanmega": "메가자리",
+    "Yveltal": "이벨타르",
+    "Zangoose": "쟝고",
+    "Zapdos": f"썬더",
```
Comment (catnet23) [inline static/locales/ko.json]: remove the f to fix the travis error.

Comment (kangsLee) [inline static/locales/ko.json]: Thank you. I changed it.

File: static/locales/ko.json [hunk 2]
```diff
@@ -0,0 +1,755 @@
+{
+    "roadmap": "지도",
+    "satellite": "인공위성",
+    "nolabels_style": "라벨없음",
+    "dark_style": "다크",
+    "style_light2": "라이트2",
+    "style_pgo": "PokemonGo",
+    "dark_style_nl": "다크(라벨없음)",
+    "style_light2_nl": "라이트2(라벨없음)",
+    "style_pgo_nl": "PokemonGo(라벨없음)",
```
Comment (sebastienvercammen) [inline static/locales/ko.json]: Should `"style_pgo_nl": "PokemonGo(라벨없음)",` not be `"style_pgo_nl": "PokemonGo (라벨없음)",` (added space)? Or is this how it's written in Korean?

Comment (kangsLee) [inline static/locales/ko.json]: need space?
I wrote it by referring to Japanese translation.
I will add a space. thanks

**Pull request merged**